Firmware/lcd.cpp

```diff
@@ -190,7 +190,7 @@ static void lcd_begin(uint8_t clear)
 	#endif
 }
 
-int lcd_putchar(char c, FILE *)
+void lcd_putchar(char c, FILE *)
```

Comment:
This function must return int, because it is passed as a function pointer into those streams - expecting to have signature of: int (*putc)(char c, FILE *) and shall return the character c if succeeded or EOF if failed. Nevertheless, the return value is not compared anywhere, so a simple "return c;" would do just fine.